cmake/modules/AddSwift.cmake

```diff
@@ -359,7 +359,7 @@ function(_add_variant_link_flags)
     endif()
   elseif("${LFLAGS_SDK}" STREQUAL "ANDROID")
     list(APPEND result
-        "-ldl" "-llog" "-latomic"
+        "-ldl" "-llog" "-latomic" "-licudata" "-licui18n" "-licuuc"
```

Comment:
My experience was these libraries are essential for doing anything with Swift Strings. Have you tried copying them across? I can revert this if you like.